### PR TITLE
Update getting-started.md

### PR DESCRIPTION
**Pull request by cpreid**
Adding examples to metric sources docs for DWH native

### CODE REVIEW

**Comment by jasonwzm** [comment]
🚀 Deployed on https://64dfabc93543ac00a142ab69--cozy-fox-0defec.netlify.app

File: docs/statsig-warehouse-native/getting-started.md [hunk 1]
```diff
@@ -31,6 +31,19 @@ _Note - while you're here, you can go to **Basic Settings** and add any Custom I
...
 ## Step 2 - Create a Metric Source
 
+### Overview
+
+A Metric Source gives you the ability to define a custom SQL query into your Data Warehouse that generates a resultset on which you'll be able to use Statsig's Metric builder to define Metrics. Typically, customer data pipelines collecting a variety of categories of data from many data sources will result in a number of tables in the data warehouse. For example:
+
+* Web Clickstream events
+* Purchase events
+* Support Events (ie; via offline and callcenter data)
+* Product Return events
```
Comment (jkw-statsig) [inline docs/statsig-warehouse-native/getting-started.md:41]: Could you make the casing consistent here? I think only the first word in each line needs to be capitalized

**Review by jkw-statsig — APPROVED** [review]

**Review by cpreid — COMMENTED** [review]
fixed casing

File: docs/statsig-warehouse-native/getting-started.md (hunk 1 above)
Comment (cpreid) [reply docs/statsig-warehouse-native/getting-started.md]: fixed

**Review by cpreid — COMMENTED** [review]
made one last critical phrasing change

File: docs/statsig-warehouse-native/getting-started.md [hunk 2]
```diff
@@ -31,6 +31,19 @@ _Note - while you're here, you can go to **Basic Settings** and add any Custom I
 
 ## Step 2 - Create a Metric Source
 
+### Overview
+
+A Metric Source gives you the ability to define a custom SQL query into your Data Warehouse that generates a resultset on which you'll be able to use Statsig's Metric builder to define Metrics. Typically, customer data pipelines collecting a variety of categories of data from many data sources will result in a number of tables in the data warehouse. For example:
+
+* Web clickstream events
+* Purchase events
+* Support events (ie; via offline and callcenter data)
```
Comment (vmadhusudanan) [inline docs/statsig-warehouse-native/getting-started.md]: call center

Comment (vmadhusudanan) [inline docs/statsig-warehouse-native/getting-started.md]: i.e.

**Review by cpreid — COMMENTED** [review]
fixed typos

**Review by cpreid — COMMENTED** [review]
done

**Review by vmadhusudanan — APPROVED** [review]

**Pull request merged**